src/pages/MusicPage.tsx

```
@@ -24,59 +25,64 @@ const MusicPage: React.FC = () => {
-                    </a>
+                <div className="relative z-10 flex flex-col md:flex-row gap-12 items-center">
+                  <div className="w-64 h-64 rounded-2xl overflow-hidden shadow-2xl border border-white/10 shrink-0">
+                    <img
+                      src={singleTrack.featured_image_src || '/images/hero-background.webp'}
+                      className="w-full h-full object-cover"
+                      alt={singleTrack.title?.rendered}
+                    />
+                  </div>
+
+                  <div className="text-center md:text-left flex-1">
+                    <h1 className="text-4xl md:text-6xl font-black font-display mb-4" dangerouslySetInnerHTML={{ __html: singleTrack.title?.rendered }} />
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)